partcad-cli/src/partcad_cli/click/commands/list/packages.py

```
@@ -1,20 +1,28 @@
+The primary purpose of this interface is to feed user interfaces like IDEs with the list of packages that are worth
+showing.
+When no recursion in requested, it shows the current package if and only if it has any parts, sketches, or assemblies.
+"""
 
 
 @click.command(help="List imported packages")
 @click.option("-r", "--recursive", is_flag=True, help="Recursively process all imported packages")
 @click.argument("package", type=str, required=False, default=".")  # help="Package to retrieve the object from"
 @click.pass_obj
-def cli(ctx, recursive, package):
+def cli(ctx: Context, recursive: bool, package: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)